docs/components/List/Web.stories.tsx

```diff
@@ -19,64 +31,294 @@ const BasicTemplate: ComponentStory<typeof List> = args => (
-    },
-    {
-      id: 2,
-      icon: "checkmark",
-      iconColor: "green",
-      title: "Josh Elford completed a visit",
-      content: "Annual Maintenance",
-      caption: "2 hours ago",
-      section: "Today",
-      onClick: () => alert("TODO: Implement onClick"),
-    },
-    {
-      id: 3,
-      icon: "badInvoice",
-      title: "Payment failed",
-      content: "For services rendered",
-      value: "$300.00",
-      caption: "1 day ago",
-      onClick: () => alert("TODO: Implement onClick"),
-      section: "Yesterday",
-    },
-  ],
+const sectionedListItems: ListItemProps[] = [
+  {
+    id: 1,
+    icon: "addNote",
+    title: "Darryl Tec added a note",
+    content: [
+      "_'Called the client. Asked if they want the luxury package, they said yes!'_",
+      "Deck Build",
+    ],
+    caption: "1 minute ago",
+    section: "Today",
+    isActive: true,
+    onClick: () => alert("TODO: Implement onClick"),
+  },
+  {
+    id: 2,
+    icon: "checkmark",
+    iconColor: "green",
+    title: "Josh Elford completed a visit",
+    content: "Annual Maintenance",
+    caption: "2 hours ago",
+    section: "Today",
+    onClick: () => alert("TODO: Implement onClick"),
+  },
+  {
+    id: 3,
+    icon: "badInvoice",
+    title: "Payment failed",
+    content: "For services rendered",
+    value: "$300.00",
+    caption: "1 day ago",
+    onClick: () => alert("TODO: Implement onClick"),
+    section: "Yesterday",
+  },
+];
+SectionedListItems.args = { items: sectionedListItems };
+
+interface SPListItemProps extends BaseListItemProps {
+  readonly name: string;
+  readonly avatarUrl: string;
+  readonly onClick?: () => void;
+}
+
+const serviceProviderListItems: SPListItemProps[] = [
+  {
+    id: 1,
+    name: "Rob Lane",
+    avatarUrl: "https://randomuser.me/api/portraits/men/1.jpg",
+    onClick: () => alert("TODO: Implement onClick"),
+  },
+  {
+    id: 2,
+    name: "Marlene Hamilton",
+    avatarUrl: "https://randomuser.me/api/portraits/women/2.jpg",
+    onClick: () => alert("TODO: Implement onClick"),
+  },
+  {
+    id: 3,
+    name: "Miriam Graves",
+    avatarUrl: "https://randomuser.me/api/portraits/women/3.jpg",
+    onClick: () => alert("TODO: Implement onClick"),
+  },
+  {
+    id: 4,
+    name: "Milton Payne",
+    avatarUrl: "https://randomuser.me/api/portraits/men/3.jpg",
+    onClick: () => alert("TODO: Implement onClick"),
+  },
+  {
+    id: 5,
+    name: "Madison Simmons",
+    avatarUrl: "https://randomuser.me/api/portraits/women/4.jpg",
+    onClick: () => alert("TODO: Implement onClick"),
+  },
+];
+
+function RenderServiceProviderList({
+  listItem,
+}: {
+  readonly listItem: SPListItemProps;
+}) {
+  return (
+    <Flex template={["shrink", "grow", "shrink"]} align="center">
+      <Avatar imageUrl={listItem.avatarUrl} />
+      <Heading level={4}>{listItem.name}</Heading>
+      <Icon name="arrowRight" size="base" color="interactive" />
+    </Flex>
```

Comment:
I'm very happy with how we ended up getting this to work!

being able to throw in a `Flex` with your layout and not fuss with any padding, hover, border and all that is really great.